src/providers/Comscore.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Comscore
+ * https://direct.comscore.com/clients/help/FAQ.aspx#faqTagging
+ *
+ * @class
+ * @extends BaseProvider
+ */
+
+class ComscoreProvider extends BaseProvider {
+  constructor() {
+    super();
+    this._key = "COMSCORE";
+    this._pattern = /sb\.scorecardresearch\.com.+(?<!\.js)$/;
+    this._name = "Comscore";
+    this._type = "marketing";
+  }
+
+  /**
+   * Retrieve the column mappings for default columns (account, event type)
+   *
+   * @return {{}}
+   */
+  get columnMapping() {
+    return {
+      account: "c2",
+      requestType: "requestType"
+    };
+  }
+
+  /**
+   * Retrieve the group names & order
+   *
+   * @returns {*[]}
+   */
+  get groups() {
+    return [
+      {
+        key: "general",
+        name: "General"
+      },
+      {
+        key: "custom",
+        name: "Custom"
+      }
+    ];
+  }
+
+  /**
+   * Get all of the available URL parameter keys
+   *
+   * @returns {{}}
+   */
+  get keys() {
+    return {
+      c1: {
+        name: "Tag Type",
+        group: "general"
+      },
+      c2: {
+        name: "Client ID",
+        group: "general"
+      }
+    };
+  }
+
+  /**
+   * Parse a given URL parameter into human-readable form
+   *
+   * @param {string}  name
+   * @param {string}  value
+   *
+   * @returns {void|{}}
+   */
+  handleQueryParam(name, value) {
+    let result = {};
+
+    if (name.startsWith("c")) {
+      result = {
+        key: name,
+        field: name,
+        value: value,
+        group: "custom"
+      };
+    } else {
+      result = super.handleQueryParam(name, value);
+    }
+    return result;
+  }
+
+  /**
+   * Parse custom properties for a given URL
+   *
+   * @param    {string}   url
+   * @param    {object}   params
+   *
+   * @returns {Array}
+   */
+  handleCustom(url, params) {
```

Comment:
I don't think the `handleCustom()` is even necessary here. Both of the parameters are static and could be mapped in `columnMapping()` (account already is).

Reply:
Can I put the value of the c1 & c2 parameters in a "General" group without using handleCustom? I would like to call them out in their own group as well as putting them in the "Custom" group since I believe they are the only constant, required parameters that do not change between clients
<img width="785" alt="Screen Shot 2019-04-30 at 8 24 44 PM" src="https://user-images.githubusercontent.com/8012764/57001353-40e43c80-6b86-11e9-990d-b29ee4e714b2.png">